frontend/components/MyGrave.tsx

```diff
@@ -52,68 +50,45 @@ export default function MyGrave() {
                     <JoinGravePanel />
                   </Flex>
                 </Box>
+                <Accordion mb={'4'} allowToggle>
+                  <AccordionItem>
+                    <h2>
+                      <AccordionButton>
+                        <Box as="span" flex="1" textAlign="left">
+                          Advanced info
+                        </Box>
+                        <AccordionIcon />
+                      </AccordionButton>
+                    </h2>
+                    <AccordionPanel pb={4}>
+                      <Text>LSP7 Universal Receiver Delegate</Text>
+                      <a
+                        href={`${constants.LUKSO_EXPLORER.TESTNET.ADDRESS}${URDLsp7}`}
+                        style={{ textDecoration: 'underline' }}
+                        target="_blank"
+                      >
+                        {URDLsp7}
+                      </a>
+                      <Text>LSP8 Universal Receiver Delegate</Text>
+                      <a
+                        href={`${constants.LUKSO_EXPLORER.TESTNET.ADDRESS}${URDLsp8}`}
+                        style={{ textDecoration: 'underline' }}
+                        target="_blank"
+                      >
+                        {URDLsp8}
+                      </a>
+                    </AccordionPanel>
+                  </AccordionItem>
+                </Accordion>
               </Box>
             </Box>
           ) : (
             <SignInBox />
           )}
         </Box>
-        <Image
-          mx={'40px'}
-          src={logoPath}
-          alt="Universal-Grave-logo"
-          width={'300px'}
-        />
+        <Image src={logoPath} alt="Universal-Grave-logo" width={'300px'} />
       </Stack>
-      {account && graveVault ? (
-        <Box>
-          <Stack mb="30px" mt="30px" direction={'row'}>
-            <Text fontSize="m" color="white" fontFamily="Bungee">
-              YOUR GRAVEYARD
-            </Text>
-            <IconButton
-              size="m"
-              aria-label={'View Graveyard'}
-              icon={<FaExternalLinkAlt />}
-              variant="ghost"
-              onClick={() => window.open(`/grave/${account}`, '_blank')}
-            />
```

Comment:
I think it's good to keep this icon along with the "share" because they know they have a dedicated page without copy/paste 

Or would that be confusing?